internal/kibana/policies.go

```diff
@@ -72,6 +72,62 @@ func (c *Client) GetPolicy(policyID string) (*Policy, error) {
 	return &resp.Item, nil
 }
 
+// GetRawPolicy fetches the given Policy with all the fields in the Ingest Manager.
+func (c *Client) GetRawPolicy(policyID string) (json.RawMessage, error) {
+	statusCode, respBody, err := c.get(fmt.Sprintf("%s/agent_policies/%s", FleetAPI, policyID))
+	if err != nil {
+		return nil, errors.Wrap(err, "could not get policy")
+	}
+
+	if statusCode != http.StatusOK {
+		return nil, fmt.Errorf("could not get policy; API status code = %d; response body = %s", statusCode, respBody)
+	}
+
+	var resp struct {
+		Item json.RawMessage `json:"item"`
+	}
+
+	if err := json.Unmarshal(respBody, &resp); err != nil {
+		return nil, errors.Wrap(err, "could not convert policy (response) to JSON")
+	}
+
+	return resp.Item, nil
+}
+
+// ListRawPolicy fetches all the Policies in the Ingest Manager.
+func (c *Client) ListRawPolicy() ([]json.RawMessage, error) {
+	itemsRetrieved := 0
+	currentPage := 1
+	var items []json.RawMessage
+	var resp struct {
+		Items   []json.RawMessage `json:"items"`
+		Total   int               `json:"total"`
+		Page    int               `json:"page"`
+		PerPage int               `json:"perPage"`
+	}
+
+	for finished := false; !finished; finished = itemsRetrieved == resp.Total {
+		statusCode, respBody, err := c.get(fmt.Sprintf("%s/agent_policies?full=true&page=%d", FleetAPI, currentPage))
+		if err != nil {
+			return nil, errors.Wrap(err, "could not get policies")
+		}
+
+		if statusCode != http.StatusOK {
+			return nil, fmt.Errorf("could not get policies; API status code = %d; response body = %s", statusCode, respBody)
+		}
+
+		if err := json.Unmarshal(respBody, &resp); err != nil {
+			return nil, errors.Wrap(err, "could not convert policies (response) to JSON")
+		}
+
+		itemsRetrieved += len(resp.Items)
+		currentPage += 1
+		items = append(items, resp.Items...)
```

Comment:
Nit. To avoid having to store all policies in memory, we could pass a visitor method here, so policies can be directly processed.
The signature of the function would be something like this:
```
func (c *Client) ForEachRawPolicy(visitFn func(json.RawMessage) error) error {
```
And then here:
```
for _, item := range resp.Items {
    if err := visitFn(item); err != nil {
        return err
    }
}
```

But no need to do it now, we can make this change later if this is a problem at some point.